packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -311,7 +311,7 @@ test('simple trades', async t => {
 });
 
 test('limit', async t => {
-  const { mintLimit, anchor } = t.context;
+  const { anchor } = t.context;
```

Comment:
for the same reasons you stopped getting `mintLimit` from the context, it should be removed from the context maker https://github.com/Agoric/agoric-sdk/blob/a1dedeae72908fda45afcb6038d76f8359adc8de/packages/inter-protocol/test/psm/test-psm.js#L129